scripts/installIos.ts

```diff
@@ -0,0 +1,57 @@
+import { join } from "https://deno.land/std@0.224.0/path/mod.ts"
+import { updateBackgroundControl } from './commons.ts'
+
+const projectPath = Deno.env.get('ADALO_APP_PROJECT_PATH') as string
+const projectName = Deno.env.get('ADALO_APP_PROJECT_NAME') as string
+
+const runPlutilCommand = async (projectName: string, args: string[]) => {
+  const target = `ios/${projectName}/Info.plist`
+  const td = new TextDecoder()
+  const options = { args: [...args, target] }
+  const proc = await new Deno.Command('plutil', options).output()
+
+  const out = td.decode(proc.stdout).trim()
+  const err = td.decode(proc.stderr).trim()
+
+  console.log('stdout:  ', out)
+  if (err) {
+    console.error('stderr:  ', err)
+  }
+
+  if (!proc.success) {
+    throw new Error(`plutil failed with exit code ${proc.code}`)
+  }
+}
+
+Deno.chdir(projectPath)
+
+const infoPlistPath = join(projectPath, `ios/${projectName}/Info.plist`)
+const infoPlistContent = await Deno.readTextFile(infoPlistPath)
+
+if (infoPlistContent.includes('<string>audio')) {
```

Comment:
could you run something like this to extract a value instead of reading the whole file as text?
`plutil -extract UIBackgroundModes raw -o - ./Info.plist`